services/parser-database/tests/test_connector.py

```diff
@@ -35,6 +35,35 @@
 }
 
 
+in_memory_value_mock_no_decimals = {
+    "ciclista": [
+        {"number": 5, "frequency": 3},
+        {"number": 45, "frequency": 6},
+        {"number": 99, "frequency": 9},
+    ],
+    "licencia": [
+        {"number": 89, "frequency": 3},
+        {"number": 45, "frequency": 3},
+        {"number": 125, "frequency": 15},
+    ],
+}
+
+
+articles_in_memory = {'5': {'wordCount': 32}, '45': {'wordCount': 40}, '89': {'wordCount': 16},
+                      '99': {'wordCount': 50}, '125': {'wordCount': 200}}
+
+
+articles_in_memory_no_wordCount = {'5': {}, '45': {}, '89': {}, '99': {}, '125': {}}
+
+
+def logn(num):
+    """
+    Mocks the natural log of a number to try to
+    minimize decimal points
+    """
+    return num
```

Comment:
nit: this is very questionable, but I'll allow it 😆

Reply:
hahhaha yeah it was my only idea on how to minimize decimal points

Reply:
maybe int(log(num)) might be a little less questionable